tempest/api/workloadmgr/restore/test_exclude_cinder_volume.py

```diff
@@ -0,0 +1,266 @@
+import json
+import os
+import sys
+import time
+import paramiko
+
+import yaml
+from oslo_log import log as logging
+
+from tempest import command_argument_string
+from tempest import config
+from tempest import reporting
+from tempest import tvaultconf
+from tempest.api.workloadmgr import base
+from tempest.lib import decorators
+from tempest.util import cli_parser
+from tempest.util import query_data
+
+sys.path.append(os.getcwd())
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+
+    credentials = ['primary']
+
+    @classmethod
+    def setup_clients(cls):
+        super(WorkloadTest, cls).setup_clients()
+
+
+    def assign_floating_ips(self, vm_id, cleanup):
+        fip = self.get_floating_ips()
+        LOG.debug("\nAvailable floating ips are {}: \n".format(fip))
+        self.set_floating_ip(str(fip[0]), vm_id, floatingip_cleanup=cleanup)
+        return(fip[0])
+
+
+    def data_ops_on_mount_point(self, flo_ip, mount_point, file_count):
+        global volumes
+        ssh = self.SshRemoteMachineConnectionWithRSAKey(str(flo_ip))
+        stdin, stdout, stderr = ssh.exec_command(
+            "sudo apt install qemu-guest-agent")
+        self.execute_command_disk_create(
+            ssh, str(flo_ip), [volumes[0]], [mount_point])
+        self.execute_command_disk_mount(
+            ssh, str(flo_ip), [volumes[0]], [mount_point])
+        self.addCustomfilesOnLinuxVM(ssh, mount_point, file_count)
+        ssh.close()
+
+
+    @decorators.attr(type='workloadmgr_cli')
+    def test_exclude_cinder_volume_backup(self):
+        try:
+            deleted = 0
+            global volumes
+            ## VM and Workload ###
+            tests = [['tempest.api.workloadmgr.restore.add_metadata_to_exclude_cinder_volume_from_backup',
+                      0],
+                     ['tempest.api.workloadmgr.restore.check_volume_status_after_restore',
+                      0]
+                     ]
+
+            reporting.add_test_script(tests[0][0])
+
+            #create key_pair value 
+            kp = self.create_key_pair(
+                tvaultconf.key_pair_name, keypair_cleanup=True)
+            LOG.debug("Key_pair : " + str(kp))
+
+            #create vm
+            vm_id = self.create_vm(key_pair=kp, vm_cleanup=False)
```

Comment:
we should set vm_cleanup = true so that it can be cleaned up after test execution. That way we can remove specific code to handle vm deletion in finally.